src/alfrescoApi.ts

```diff
@@ -68,7 +68,12 @@ export class AlfrescoApi {
             config = {};
         }
 
+        if (config.domainPrefix) {
+            Storage.setPrefix(config.domainPrefix);
```

Comment:
Is this method static intentionally? I don't know if it is a valid use case at all, but wouldn't it cause problem if I want to use 2 different js-api with different domainPrefix? With this, the second one would overwrite the first's settings.